### ISSUE TITLE
Python 3 Support

### ISSUE THREAD

**Issue by willtheorangeguy** [minimaxir/facebook-page-post-scraper#8]
I am trying to install this on Python 3 and it doesn't seem to be working. 
Is this version for Python 3 or 2 or am I installing it incorrectly.

**Comment by marcogoldin** [comment]
It's not a package on pypi, it's a python script, also optimized for 2.7, so for Python 3 you'll have to work on some changes.

**Comment by willtheorangeguy** [comment]
So I should just be able to run it right out of the box? (I have 2.7 as backup for some of these kinds of scripts.)

Maybe we can work on making it work with Python 3.

**Comment by marcogoldin** [comment]
Yes you should, i was planning to port it on python 3 me too. I guess it should be enough to drop urrlib2 and replace:

`import urllib2 > import urllib.request
req = urllib2.Request(url) > urllib.request.Request(url)
response = urllib2.urlopen(req) > response = urllib.request.urlopen(req)
return response.read() > return response.read().decode('utf-8')
`

and check for print functions to wrap them in brackets, or maybe add some.

**Comment by minimaxir** [comment]
Yes, those two areas should only be the changes needed to get this working in Python 3.

**Comment by willtheorangeguy** [comment]
Yep, we can change that.

**Comment by willtheorangeguy** [comment]
I can fix the brackets around the print statements.

**Comment by minimaxir** [comment]
That being said, if you are making a PR, please submit the Python 3 version as a separate script.

**Comment by willtheorangeguy** [comment]
Sure. I'll start now.

**Comment by willtheorangeguy** [comment]
I finished making the changes and submitted a Pull Request.

**Comment by marcogoldin** [comment]
Hi everyone, it's been a while but i had some spare time and checked out on this Python 3 compatibiliy.  
@willtheorangeguy, dealing with print statements is not enough for the script to work. 
I fixed all the problems, urllib, error handling, and most importantly added StringIO, otherwise it would have throws errors caused by string bytes format. Checked in Python 3.5.2, perfectly working.
But i have a 403 denied access when i try to git push to the master. Why is that? Should i being added to the repo contributors for a quick update?
